packages/web/src/scss/components/UNSTABLE_Section/index.html

```diff
@@ -0,0 +1,3 @@
+<section class="UNSTABLE_Section">
+
+</section>
```

Comment:
Should't be something inside the section?

Reply:
Added.